setup.py

```diff
@@ -281,7 +280,10 @@ def install_requirements(configure):
                 ],
             }[server_mode]
     requirements += {
-            u'Y': [u'-r', u'requirements/tests.txt'],
+            u'Y': [
+                u'-r', u'requirements/tests.txt',
+                u'-r', u'poleno/dummymail/requirements.txt'
```

Comment:
@martinmacko47 Ako to poriesit s requirements? Mam pridat dalsi subor alebo vsetky requirements pre testy mam napisat do jedneho suboru `requirements/tests.txt`
`localmail==0.3` nie som schopny nainstalovat, skonci na chybe 
```
...
import twisted.plugins as package
ImportError: No module named plugins
```
, momentalne mi nainstaloval verziu `0.5.0`

Reply:
@viliambalaz Aha, to mi nedoslo, ze testy v `poleno/dummymail` pouzivaju `localmail`, ale mod `local_with_no_mail` ho nenainstaluje. Skus pri testoch pouzit mod `local_with_local_mail`. Ten includne `requirements/local_mail.txt`, ktory nainstaluje aj `localmail`.

Co sa tyka verzie `localmail==0.3`. Nenainstlaovalo ti to lokalne na tvojom pocitaci, alebo na githube? Mne lokalne to `localmail==0.3` zvladlo nainstalovat, ked som pouzil mod `local_with_local_mail`.

Reply:
:ok: podarilo sa mi nainstaloval `localmail==0.3` aj na githube aj u seba.